webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const webpack = require('webpack');
+const path = require('path');
+
+module.exports = {
+  mode: 'development',
+  devtool: 'inline-source-map',
+  entry: './src/index.ts',
+  resolve: {
+    extensions: ['.ts', '.tsx', '.js'],
```

Comment:
I'm not sure we should resolve `.js` extensions with `.ts` file
but probably we want serve `.json` files for future

Reply:
Without js does not work for some dependencies